### PR TITLE
Add banner to homepage

### PR DESCRIPTION
**Pull request by harshbutfairx**
Desktop:
Screenshot 2024-06-09 at 21 23 23

Tablet:
Screenshot 2024-06-09 at 21 24 12

Mobile:
IMG_0619

### CODE REVIEW

**Comment by pranay01** [comment]
@harshbutfairx any idea why preview is failing?

**Comment by YounixM** [comment]
@pranay01 : It's passing now.

@harshbutfairx : The font size of the banner text seems to be bigger than the content on the website

File: components/Banner/Banner.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+const Banner = () => {
+    const [visible, setVisible] = useState(true);
+    const pathname = usePathname();
+    const [isHomePage, setIsHomePage] = useState(false);
+
+    useEffect(() => {
+        setIsHomePage(pathname === '/');
+    }, [pathname]);
+
+    if (!visible || !isHomePage) return null;
```
Comment (YounixM) [inline components/Banner/Banner.tsx:17]: Do we want to show the banner in only the homepage ?

**Comment by harshbutfairx** [comment]
> @pranay01 : It's passing now.
> 
> @harshbutfairx : The font size of the banner text seems to be bigger than the content on the website

Hi Yunus, this is intentional and as per the design spec.

File: components/Banner/Banner.tsx (hunk 1 above)
Comment (harshbutfairx) [reply components/Banner/Banner.tsx:17]: Yes, this is intentional. The banner is supposed to be only visible on the homepage.

**Pull request merged**